### ISSUE TITLE
Simpler way of creating locks from multiple requirements

### ISSUE THREAD

**Issue by cm-tm** [sbg/dante#2]
If I have multiple requirements files and packages from all of them installed in the current environment, then running `dante lock -s` would lock all packages in `lock_file_path`, even those that are listed in a different requirement file. So instead I need to perform multiple lock calls:

```
dante lock -s -r requirements.txt -f requirements.lock
dante lock -s -r requirements-dev.txt -f requirements-dev.lock
...
```

With the existing support for multiple requirements files and multiple lock files, wouldn't it be better if there was a clear relation between each requirement file and a lock file so that `dante lock -s` would **modify multiple lock files at once**?

**Comment by QuicksilverMachine** [comment]
I agree, I noticed the same issue when using it myself. I'll take a look at it when I get the chance. Any suggestion is welcome.

**Comment by 1buran** [comment]
Ping..
